plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineSdkClasspathContainer.java

```diff
@@ -0,0 +1,59 @@
+package com.google.cloud.tools.eclipse.appengine.facets;
+
+import java.io.File;
+
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.jdt.core.IClasspathContainer;
+import org.eclipse.jdt.core.IClasspathEntry;
+import org.eclipse.jdt.core.JavaCore;
+
+import com.google.cloud.tools.appengine.cloudsdk.PathResolver;
+import com.google.common.annotations.VisibleForTesting;
+
+public final class AppEngineSdkClasspathContainer implements IClasspathContainer {
+
+  // TODO should be changed once app-tools-lib can provide the directory
```

Comment:
actually this is fixed. I forgot to close the bug. We just need to upgrade to 0.1.2.